README.md

```
@@ -32,7 +32,8 @@
 - [5. Results](#5-results)
 - [6. Conclusions](#6-conclusions)
 - [7. Recomendations](#7-recomendations)
-- [8. References](#8-references)
+- [8. Acknowledgments](#8-acknowledgments)
+- [9. References](#8-references)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)